www/src/pages/ar/faq.rtl.md

```diff
@@ -0,0 +1,58 @@
+---
+title: الأسئلة الشائعة
+description: الأسئلة المتكررة حول إنشاء تطبيق T3
+layout: ../../layouts/docs.astro
+---
```

Comment:
```suggestion
lang: ar
dir: rtl
---
```

Reply:
The suggested changes had been committed [Here](https://github.com/t3-oss/create-t3-app/commit/ede71d91fbc11ceb56a1b9d13c7f0fee2e9f0b5f)

added
```
lang: ar
dir: rtl
```